service/reportsStorage/storage_test.go

```diff
@@ -0,0 +1,659 @@
+package reportsStorage
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/golang/mock/gomock"
+	"github.com/rs/zerolog"
+
+	"github.com/iryonetwork/wwm/gen/reports/models"
+	storageModels "github.com/iryonetwork/wwm/gen/storage/models"
+
+	"github.com/iryonetwork/wwm/storage/s3/mock"
```

Comment:
could not import github.com/iryonetwork/wwm/storage/s3/mock (cannot find package "github.com/iryonetwork/wwm/storage/s3/mock" in any of:
	/tmp/tmp.qkYKz9nbyP/src/github.com/iryonetwork/wwm/vendor/github.com/iryonetwork/wwm/storage/s3/mock (vendor tree)
	/usr/local/go/src/github.com/iryonetwork/wwm/storage/s3/mock (from $GOROOT)
	/tmp/tmp.qkYKz9nbyP/src/github.com/iryonetwork/wwm/storage/s3/mock (from $GOPATH))